test/connectors/GmpBulkConnectorSpec.scala

```diff
@@ -22,6 +22,7 @@ import models._
 import org.mockito.Matchers
 import org.mockito.Mockito._
 import org.scalatest.BeforeAndAfter
+import org.scalatest.concurrent.ScalaFutures.convertScalaFuture
```

Comment:
instead of this, you can simply do extend ScalaFutures `with ScalaFutures`

Reply:
done